### PR TITLE
Add basic functional tests

### PR DESCRIPTION
**Pull request by mnapoli**
Here are 2 functional tests to cover that:

- Symfony can be deployed without a cache, and it should work
- Symfony can be deployed with a prewarmed compiled container, and it should work

I run both in a Docker container using the official Bref images, with the test project mounted as readonly.

Without the `BrefKernel` (i.e. the stock Symfony app):

- ❌  Symfony can be deployed without a cache, and it should work (tries to write on the readonly `var` directory)
- ✅  Symfony can be deployed with a prewarmed compiled container, and it should work

With the `BrefKernel`:

- ✅  Symfony can be deployed without a cache, and it should work
- ✅  Symfony can be deployed with a prewarmed compiled container, and it should work

These tests should help with any refactoring moving forward.

### CODE REVIEW

File: tests/Functional/FunctionalTest.php [hunk 1]
```diff
@@ -0,0 +1,79 @@
...
+{
+    public function test Symfony works with an empty cache(): void
+    {
+        $this->composerInstall();
+        $this->clearCache();
+        $symfonyConsole = $this->runSymfonyConsole();
+        $this->assertTrue($symfonyConsole->isSuccessful());
+    }
+
+    public function test Symfony works with a compiled container(): void
```
Comment (Nemo64) [inline tests/Functional/FunctionalTest.php:19]: spaces in method names? 🤨

Comment (mnapoli) [reply tests/Functional/FunctionalTest.php:19]: https://mnapoli.fr/using-non-breakable-spaces-in-test-method-names/ 😉 

Used in the tests in Bref, for example https://github.com/brefphp/bref/blob/master/tests/Functional/FpmRuntimeTest.php

File: tests/Functional/App/config/packages/cache.yaml [hunk 2]
```diff
@@ -0,0 +1,19 @@
+framework:
+    cache:
+        # Unique name of your app: used to compute stable namespaces for cache keys.
```
Comment (Nyholm) [inline tests/Functional/App/config/packages/cache.yaml:3]: Some comments here could be removed

Comment (mnapoli) [reply tests/Functional/App/config/packages/cache.yaml:3]: 👍  I simply created a new Symfony project and did not change anything in there.

**Pull request merged**